websites/F/Flightradar24/presence.ts

```diff
@@ -0,0 +1,690 @@
+const presence = new Presence({
+		clientId: "930914836773224498"
+	}),
+	browsingStamp = Math.floor(Date.now() / 1000);
+let remaining = 0,
+	leftTimestamp = 0,
+	player = {
+		total: "",
+		elapsed: "",
+		isPlaying: false
+	},
+	video = {
+		currentTime: 0,
+		duration: 0,
+		paused: true
+	};
+
+presence.on(
+	"iFrameData",
+	(data: {
+		elapsed: string;
+		total: string;
+		isPlaying: boolean;
+		currentTime: number;
+		duration: number;
+		paused: boolean;
+	}) => {
+		(player = {
+			total: data.total,
+			elapsed: data.elapsed,
+			isPlaying: data.isPlaying
+		}),
+			(video = {
+				currentTime: data.currentTime,
+				duration: data.duration,
+				paused: data.paused
+			});
+	}
+);
+
+presence.on("UpdateData", async () => {
+	const [elapsed, buttons, images, left] = await Promise.all([
+			presence.getSetting<boolean>("elapsed"),
+			presence.getSetting<boolean>("buttons"),
+			presence.getSetting<boolean>("images"),
+			presence.getSetting<boolean>("left")
```

Comment:
`left`, what does that mean?

Reply:
> `left`, what does that mean?

![NVIDIA_Share_ox4VHXLccj](https://user-images.githubusercontent.com/44418429/154379421-3b4b549e-3afd-4dea-97c7-bd6c06f84e7c.png)